modules/ipi-install-replacing-the-bare-metal-master-node.adoc

```diff
@@ -0,0 +1,154 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc
+
+:_content-type: PROCEDURE
+[id='replacing-the-bare-metal-master-node_{context}']
+= Replacing the bare metal master node
+
+Use the following procedure to replace an installer-provisioned {product-title} cluster.
+
+[IMPORTANT]
+====
+When the replacement BareMetalHost is created from a new physical machine or an incomplete prior YAML file configuration, the `externallyProvisioned:` field in the configuration file must be set to `false` since the machine has not been provisioned.
```

Comment:
Maybe "since the machine has not been provisioned" -> "to allow the cluster to provision the host. Existing control plane BareMetalHosts may have the externallyProvisioned flag set to true if they were provisioned by the OpenShift Installer."

This could be too much information. Just thinking about how to alert users that when they need to think about this is if copying/editing an existing host.

Reply:
Added Additional Resource section with Bare Metal Configuration content.